covid_graphs/covid_graphs/scripts/server.py

```diff
@@ -25,13 +28,14 @@
 covid19_normal_app = CountryReport.create_dashboard(args.data_dir, server, GraphType.Normal)
 covid19_semilog_app = CountryReport.create_dashboard(args.data_dir, server, GraphType.SemiLog)
 covid19_loglog_app = CountryReport.create_dashboard(args.data_dir, server, GraphType.LogLog)
-covid19_heatmap_app = HeatMap(args.simulated_polynomial).create_app(server)
-covid19_heatmap_exponential_app = HeatMap(args.simulated_exponential).create_app(server)
+# covid19_heatmap_app = HeatMap(args.simulated_polynomial).create_app(server)
+# covid19_heatmap_exponential_app = HeatMap(args.simulated_exponential).create_app(server)
```

Comment:
These were forgotten I guess. I can send you the protobufs, so that you don't have to uncomment these.
```suggestion
covid19_heatmap_app = HeatMap(args.simulated_polynomial).create_app(server)
covid19_heatmap_exponential_app = HeatMap(args.simulated_exponential).create_app(server)
```

Reply:
Thanks, missed that :)